examples/reference-entity-model/src/main/ml-config/security/users/pari.json

```diff
@@ -3,10 +3,6 @@
   "description": "Pari is a MarkLogic expert",
   "password": "%%testPassword%%",
   "role": [
-    "data-hub-developer",
-    "hub-central-downloader",
-    "hub-central-load-writer",
-    "hub-central-mapping-writer",
-    "hub-central-flow-writer"
+    "hub-central-developer"
```

Comment:
Pari should have data-hub-developer right and data-hub-developer should inherit hub-central-developer?

Reply:
No. hub-central-developer grants access to the Hub Central UI. Having having access to Hub Central shouldn't be granted via the data-hub-developer role.

Reply:
ok makes sense. Although, I would still expect Pari a MarkLogic expert to have data-hub-developer role.

Reply:
I missed this - pari should still have data-hub-developer, in addition to hub-central-developer, as she'll be doing things like running hubDeploy. 

Perhaps HCD should inherit DHD by default? Is there really a use case for someone to be an HCD but not a DHD?